tornado_jsonapi/resource.py

```diff
@@ -187,23 +224,23 @@ def __enter__(self):
         def __exit__(self, type_, value, traceback):
             if self.transaction:
                 if type_ is None:
-                    yield self.connection.execute('COMMIT')
+                    yield self.connection.execute("COMMIT")
                 else:
-                    yield self.connection.execute('ROLLBACK')
+                    yield self.connection.execute("ROLLBACK")
             self.pool.putconn(self.connection)
 
     connection = yield pool.getconn(ping=False)
     if transaction:
-        yield connection.execute('BEGIN')
+        yield connection.execute("BEGIN")
     return wrapper(pool, connection, transaction)
 
 
 class DBAPI2Resource(Resource):
```

Comment:
I guess for your changes to be consistent, you'd also have to append blacklist and paging support to `DBAPI2Resource`.

Reply:
Complete. I am not sure if limit #,# is supported for all platforms.